include/common/core/memory.hpp

```diff
@@ -288,6 +307,57 @@ __XETLA_API void xetla_prefetch_global(Ty* p, uint64_t offset = 0) {
       gpu::xetla::detail::get_cache_hint(L2H)>((T*)p + (offset / sizeof(T)));
 }
 
+/// simd<T, N> block_load(const T* ptr, size_t byte_offset,
```

Comment:
is this modification need to be controlled by the different compiler version ?

Reply:
In order to support ipex, xetla has been upgraded to oneapi 2024.1. The old version is no longer supported.